readme.md

```diff
@@ -283,3 +283,26 @@ Make sure you copy all dlls from `distribution\install\app\$(Configuration);` to
 > **_NOTE:_** MeshLib distribution has x64 build only.
 
 > **_NOTE:_** Distribution is built with `_ITERATOR_DEBUG_LEVEL=0` macro defined in Debug configuration so you will need to setup this for your projects.
+
+### Distribution for .NET
+You can download [zip-archive MeshLibDotNetDist.zip](https://github.com/MeshInspector/MeshLib/releases) containing dynamic lybraries (.dll) of both MeshLib and dependant third-party libraries, and make them available during your project building and runtime execution as follows.
+
+1. Create a new C# project.
+2. Double-click on the project and open `.csproj` file
```

Comment:
in text editor? or can be setup in Visual studio? if so better to describe this way